tests/end_to_end/workflow/private_attributes_flow.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+pass
+import numpy as np
+import logging
+from openfl.experimental.workflow.interface import FLSpec, Aggregator, Collaborator
+from openfl.experimental.workflow.runtime import LocalRuntime
+from openfl.experimental.workflow.placement import aggregator, collaborator
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+class TestFlowPrivateAttributes(FLSpec):
+    """
+    Testflow to validate Aggregator private attributes are not accessible to collaborators
```

Comment:
Can we mention that in this test, private attributes are initialized by callable